### DISCUSSION TITLE
忽略udp请求

### DISCUSSION THREAD

**Discussion by johnnyhansen-away** [xjasonlyu/tun2socks#209] [Q&A]
如何只处理某一ip的tcp流量, 针对ip或端口忽略 所有udp请求

**Comment by xjasonlyu** [comment]
可以用iptables丢弃某一ip下的所有UDP packets。

**Reply by johnnyhansen-away** [reply]
> 可以用iptables丢弃某一ip下的所有UDP packets。

我是在Mac上使用的route 添加的路由, 可以指定只选TCP协议的流量么, mac下iptables怎么用能举个例子么, 谢谢
